### PR TITLE
Task30

### PR DESCRIPTION
**Pull request by yana-may25**
So I figured you just have to wrap the code in a function to make this visible, and didn't bother with creating a div inside JS as you told me, the same applies for task 29

### CODE REVIEW

File: src/task30/task30.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+function insideTheBox() {
+    let cx = document.querySelector("canvas").getContext("2d");
```
Comment (MrOlegKravchenko) [inline src/task30/task30.js]: Move out as much variables as you find reasonable to be global.

**Comment by MrOlegKravchenko** [comment]
Looks good enough.
But make the ball much smaller and the box a little bigger, e.g. square.
Also move the ball at some angle to make its movements more chaotic.

**Comment by yana-may25** [comment]
https://user-images.githubusercontent.com/75950794/229100452-7d2d3eda-85f0-4ade-87ac-f05b9f37d2a5.mp4

File: src/task30/task30.js (hunk 1 above)
Comment (yana-may25) [reply src/task30/task30.js]: but that's the point! I have wrapped the code in to the function, because idk how to make html file see the js otherwise

**Review by MrOlegKravchenko — APPROVED** [review]

**Pull request merged**